source/MRMesh/MRMeshLoadObj.cpp

```diff
@@ -143,141 +146,127 @@ tl::expected<std::vector<NamedMesh>, std::string> fromSceneObjFile( const char*
 
     Timer timer( "split by lines" );
     std::vector<size_t> newlines{ 0 };
-    tbb::enumerable_thread_specific<std::vector<size_t>> newlinesPerThread;
-    tbb::parallel_for( tbb::blocked_range<size_t>( 0, size ), [&] ( const tbb::blocked_range<size_t>& range )
+    // TODO: parallelize
```

Comment:
It looks like this place was parallel before the change. Will the performance decrease?